pkg/renderer/kubernetes_helm.go

```diff
@@ -65,6 +67,21 @@ func RenderSensorTLSSecretsOnly(values charts.MetaValues, certs *sensor.Certs) (
 	return out.Bytes(), nil
 }
 
+// overwriteEmptyImageValues overrides an empty ImageTag in charts.MetaValues to avoid errors when rendering charts.
+// This should only be used when rendering TLS charts, because `ImageTag` will never be needed there. Hence, the
+// tag value that makes explicit that this value should never appear in a user chart. More info on the issue
+// refer to ROX-16212.
+//
+// Ideally, this should not be needed, and it only happens due to the logic necessary to build manifest bundles.
+// Either the logic to render TLS secrets should be decoupled from the manifest bundle, and only the necessary
+// are files rendered. Or we fix the logic in the manifest bundle so that ImageTag is always set, regardless of the
+// cluster configuration.
+func overwriteEmptyImageValues(values *charts.MetaValues) {
```

Comment:
How about
```suggestion
func fixEmptyImageTag(values *charts.MetaValues) {
```
?
Optional and up to you.

Reply:
Done